test/spec/f16.wast

```diff
@@ -147,3 +154,36 @@
     (v128.const i16x8 0x5140 0xfe00  0x7c00 0x3e00 0      0x3c00 0      0x3c00))
     ;;                nan    -nan    inf    1.5    0      1      1      1
     (v128.const i16x8 0x7e00 0xfe00  0x7c00 0x3e00 0      0x3c00 0x3c00 0x3c00))
+
+;; unary arithmetic
```

Comment:
Some of these tests result in  `-nan`. Should I be avoiding those since they're non-deterministic?

Reply:
We support using `nan:canonical` and `nan:arithmetic` and `nan:<payload>` constants in spec tests, so perhaps that is an option as well. Although that might not support f16 constants out of the box, and either the interpreter or the wast runner seems to have some bugs in this area, so that may also be a problem.

If it's a pain, I'd say don't worry about changing it for now.